piwik/content.md

```diff
@@ -0,0 +1,71 @@
+# Piwik
+
+[![Build Status](https://travis-ci.org/piwik/docker-piwik.svg?branch=master)](https://travis-ci.org/piwik/docker-piwik)
+
+Piwik is the leading open-source analytics platform that gives you more than just powerful analytics:
+ - Free open-source software
+ - 100% data ownership
+ - User privacy protection
+ - User-centric insights
+ - Customisable and extensible
+
+%%LOGO%%
+
+# How to use this image
+
+The easiest is to use our `docker-compose.yml`.
+
+Make sure you have [docker-compose](http://docs.docker.com/compose/install/) installed. And then:
+
+```
+git clone https://github.com/piwik/docker-piwik.git
```

Comment:
`git clone` seems heavy-handed for just a few config files and a compose file; all of which should be fairly constant from this point forward. Could we instead just document this process in the readme?